### PR TITLE
Add LDAP loadbalancer image

### PR DESCRIPTION
**Pull request by osfrickler**

### CODE REVIEW

**Comment by berendt** [comment]
@osfrickler Ready for review?

**Comment by osfrickler** [comment]
I'm still pondering how to best handle the SSL part, that might need amending the CMD. I'm also not to sure whether EXPOSE just for the ldaps port is good, maybe not using EXPOSE and leaving that to the container deployment would be better?

If you have other review comments already, go ahead.

File: lloadd/Containerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+COPY --from=builder /usr/local/libexec/lloadd /lloadd
+
+RUN /lloadd -VV || true
+
+ENTRYPOINT ["/lloadd"]
+CMD ["-d0"]
+
+EXPOSE 636
+
+LABEL "org.opencontainers.image.documentation"="https://docs.osism.tech" \
```
Comment (berendt) [inline lloadd/Containerfile:41]: Use new docs URL

File: lloadd/Containerfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+FROM debian:bookworm
+
+RUN apt-get update && apt-get -f -y install libevent-core-2.1-7 libevent-extra-2.1-7 libsasl2-2
+
+COPY --from=builder /usr/local/etc /usr/local/etc/
+COPY --from=builder /usr/local/lib /usr/local/lib/
+COPY --from=builder /usr/local/libexec/lloadd /lloadd
+
+RUN /lloadd -VV || true
```
Comment (berendt) [inline lloadd/Containerfile:34]: Why is this required?

File: lloadd/Containerfile [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    libdebhelper-perl libedit2 libelf1 libevent-2.1-7 libfile-stripnondeterminism-perl libgmp-dev libgmpxx4ldbl \
+    libgnutls-dane0 libgnutls-openssl27 libgnutls28-dev libgnutlsxx30 libgssapi3-heimdal \
+    libhcrypto5-heimdal libhdb9-heimdal libheimbase1-heimdal libheimntlm0-heimdal libhx509-5-heimdal \
+    libicu72 libidn2-dev libkadm5clnt7-heimdal libkadm5srv8-heimdal libkafs0-heimdal \
+    libkdc2-heimdal libkrb5-26-heimdal libltdl-dev libltdl7 libmagic-mgc libmagic1 libodbc2 libodbccr2 \
+    libodbcinst2 libotp0-heimdal libp11-kit-dev libperl-dev libpipeline1 libpkgconf3 \
+    libroken19-heimdal libsasl2-dev libsl0-heimdal libsub-override-perl libtasn1-6-dev libtool libuchardet0 \
+    libunbound8 libwind0-heimdal libwrap0 libwrap0-dev libxml2 m4 man-db nettle-dev \
+    openssl pkg-config pkgconf pkgconf-bin po-debconf sensible-utils unixodbc-common unixodbc-dev \
+    && wget http://deb.debian.org/debian/pool/main/o/openldap/openldap_2.6.6+dfsg.orig.tar.xz \
```
Comment (berendt) [inline lloadd/Containerfile:20]: Use VERSION for the version and do not hardcode it.

**Review by berendt — CHANGES_REQUESTED** [review]

File: lloadd/Containerfile (hunk 1 above)
Comment (osfrickler) [reply lloadd/Containerfile:41]: wouldn't it be better to change this for all containers in one patch in order to be consistent?

File: lloadd/Containerfile (hunk 2 above)
Comment (osfrickler) [reply lloadd/Containerfile:34]: it is not required, I just copied from the dnsdist role, can be dropped instead
